.gitignore

```diff
@@ -1 +1 @@
-node_modules
+node_modules
```

Comment:
Nitpick: Please set up your editor to add newlines to end of files, you can look up the many reasons on Google for this. It will also remove any changes from this entire file and make it easy to compare your actual changes/additions.